add-permissions.js

```diff
@@ -80,6 +80,11 @@ module.exports = class AwsAddLambdaAccountPermissions {
               throw new Error(`Roles must have unique names [${name}]`);
             }
 
+            let stsAction = 'sts:AssumeRole';
+            if (allowTagSession) {
+              stsAction = ['sts:AssumeRole', 'sts:TagSession'];
+            }
```

Comment:
If we decide to keep `allowTagSession`
```suggestion
            const stsAction = ['sts:AssumeRole'];
            if (allowTagSession) {
              stsAction.push('sts:TagSession');
            }
```

Reply:
Lol, I wrote that orginally, but then switched to reduce drift in existing tests.

Reply:
If that's the goal, I'd use a ternary below, like
```js
{
  Action: allowTagSession
    ? ['sts:AssumeRole', 'sts:TagSession']
    : 'sts:AssumeRole'
}
```

Reply:
Sure. Accomplishes the same result.  I've never personally been a fan of that operator.  Seen it abused too many times.